webhook.go

```diff
@@ -202,7 +202,13 @@ func main() {
 func hookHandler(w http.ResponseWriter, r *http.Request) {
 
 	// generate a request id for logging
-	rid := uuid.NewV4().String()[:6]
+	uuid, err := uuid.NewV4()
+
+	if err != nil {
+		log.Printf("[%s] error generating uuid %s\n", uuid, err)
```

Comment:
If we can't generate a UUID, we should return an internal server error similar to [L271-L273](https://github.com/adnanh/webhook/blob/fc0544e4a2f0b5d64359d2005bbf77c8fd15aa69/webhook.go#L271-L273).